deny.toml

```
@@ -28,6 +28,8 @@ allow = [
     # Blanket addition for Unicode-3.0 libs for optional features (affects many deps, but it's a very clean license)
     # https://github.com/unicode-org/icu4x/blob/main/LICENSE
     "Unicode-3.0",
+    # it's hard to avoid this dependency here :)
```

```suggestion
    # Pulled in via aws_lc_rs when using rustls-tls and aws-lc-rs features
    # https://openssl-library.org/source/license/index.html
```
for future archeology